backend/config.json

```diff
@@ -3,5 +3,22 @@
   "PORT": 5000,
   "NUM_THREADS": 50,
   "DB_ENGINE": "mysql",
-  "DB_CHARSET": "utf8mb4"
+  "DB_CHARSET": "utf8mb4",
+  "MAX_LANGUAGES_COUNT": 8,
```

Comment:
Another question about this config. Why not to put everything (maybe except PORT) into constants in the code? The code will be more readable in that case.

What is the benefit of using json?